src/modules/setup/setupTSConfig.ts

```diff
@@ -24,19 +25,21 @@ const getTSConfig = async () => {
 }
 
 export const setupTSConfig = async (manifest: Manifest) => {
-  const tsconfigsFromBuilder = await getTSConfig()
-  if (!tsconfigsFromBuilder) {
-    return
-  }
+  log.info(`Setup TSConfig`)
+  try {
+    const tsconfigsFromBuilder = await getTSConfig()
+    if (!tsconfigsFromBuilder) {
+      log.warn(`No builders candidates for TSConfig setup`)
```

Comment:
I don't think this warning will be useful for people not developing an app with `react` or `node` builders that only run `vtex setup` (with all implicitly defined).

```suggestion
      log.warn(chalk.dim(`No builders candidates for TSConfig setup`))
```

Reply:
Do you think we should remove this log?

Reply:
can we show it only if the flag `--tsconfig` was explicitly passed?

Reply:
Yeah, this way is better